test/neural_net/style_transfer_testing_utils.hpp

```diff
@@ -0,0 +1,28 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h>
+
+#include <boost/property_tree/ptree.hpp>
+
+@interface NeuralNetStyleTransferUtils : NSObject
```

Comment:
Not sure what's going on here. I'm wondering if we need some design review before we proceed further....

Reply:
I commented above on how I was loading the golden set for testing into this testing scaffolding. These functions help with that.